.github/workflows/publish-a-package.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish a Package from Tag
+
+on:
+  push:
+    tags:
+      - 'opentelemetry-+==1.+'
+
+jobs:
+  publish:
+    name: Publish package from tag ${{ env.GITHUB_REF }}
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.7'
```

Comment:
Can this be a more current version?

Reply:
I did this to copy `./scripts/build.sh`, but I used `3.9` on my own repo and everything worked fine so I updated this value to that.